README.md

```diff
@@ -125,37 +207,19 @@ location / {
   }
 ```
 
-## Data persistence
+## C. How to persist data
 
-If you wish your OpenCTI data to be persistent while in production, you should be aware of the `volumes` section for `ElasticSearch`, `MinIO`, `Redis` and `RabbitMQ` services in the `docker-compose.yml`.
+The default for OpenCTI data is to be persistent.
```

Comment:
I believe this should be the default (when not in dev). More often than not, the user will want to persist their data, and will likely be very sad if they lost it because their container shut down.